diff --git a/dist/components/AvatarImg.svelte b/dist/components/AvatarImg.svelte
index 62697deab3e67c64114cf7cd69c1c4bf219290db..c1300efab6175709cf94fe684bbd0a3eea8b4820 100644
--- a/dist/components/AvatarImg.svelte
+++ b/dist/components/AvatarImg.svelte
@@ -5,4 +5,4 @@ export let height;
 export let style = "";
 </script>
 
-<img {src} {alt} {width} {height} {style} />
+<img {src} {alt} {width} {height} {style} loading="lazy" />
diff --git a/dist/components/EmbeddedTweet.svelte b/dist/components/EmbeddedTweet.svelte
index 68c2f2e770effdaae4a24f44ba11891a2980891f..76ff2fcd1478c9c4c57e1208a8cf168a72df5528 100644
--- a/dist/components/EmbeddedTweet.svelte
+++ b/dist/components/EmbeddedTweet.svelte
@@ -9,7 +9,6 @@ import TweetActions from "./TweetActions.svelte";
 import TweetReplies from "./TweetReplies.svelte";
 export let tweet;
 export let components = {};
-console.info(`using tweet ${JSON.stringify(tweet)}`);
 let enrichedTweet;
 try {
   enrichedTweet = enrichTweet(tweet);
diff --git a/dist/components/MediaImg.svelte b/dist/components/MediaImg.svelte
index c4cb5218112742d52aabd391d6443e19eba3ca7e..32e1ad964c3dab0ae743987b6b4284e4a8166f5a 100644
--- a/dist/components/MediaImg.svelte
+++ b/dist/components/MediaImg.svelte
@@ -4,4 +4,4 @@ export let className = "";
 export let draggable = false;
 </script>
 
-<img src={src} alt={alt} class={className} draggable={draggable} />
+<img src={src} alt={alt} class={className} draggable={draggable} loading="lazy" />
